bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/DimmerHandler.java

```diff
@@ -87,10 +85,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void handleFeedback(RadioRAFeedback feedback) {
-        if (feedback instanceof LocalZoneChangeFeedback) {
-            handleLocalZoneChangeFeedback((LocalZoneChangeFeedback) feedback);
-        } else if (feedback instanceof ZoneMapFeedback) {
-            handleZoneMapFeedback((ZoneMapFeedback) feedback);
+        if (feedback instanceof LocalZoneChangeFeedback feedbackCommand) {
+            handleLocalZoneChangeFeedback(feedbackCommand);
+        } else if (feedback instanceof ZoneMapFeedback feedbackCommand) {
+            handleZoneMapFeedback(feedbackCommand);
```

Comment:
```suggestion
        if (feedback instanceof LocalZoneChangeFeedback localZoneChangeFeedback) {
            handleLocalZoneChangeFeedback(localZoneChangeFeedback);
        } else if (feedback instanceof ZoneMapFeedback zoneMapFeedback) {
            handleZoneMapFeedback(zoneMapFeedback);
```

(or similar, it doesn't seem related to commands?)